tests/wpunit/ConfigTest.php

```diff
@@ -75,4 +75,43 @@ public function should_reset() {
 			$this->assertInstanceOf( \RuntimeException::class, $e );
 		}
 	}
+
+	/**
+	 * @test
+	 */
+	public function should_add_group_paths() {
+		Config::add_group_path( 'my-group-path-1', [ 'root' => dirname( dirname( __DIR__ ) ) . '/src/feature-1', 'relative' => 'app-1' ] );
```

Comment:
From what I gather, the second parameter of `Config::add_group_path` is an array that has to  define the `root` and `relative` keys. 

Why not change the method signature to something clearer like this?

```php
Config::add_group_path(string $group_slug, string $root, string $relative)
```